addons/transifex/models/models.py

```diff
@@ -4,9 +4,7 @@
 from odoo import models
 
 
-class BaseModel(models.AbstractModel):
-    _name = "base"
```

Comment:
I see a few manual fixes for `base`, is the script not able to fix it?

Reply:
I do not modify the base automatically. It is a choice to avoid certain unwanted changes.